Refactor contributing (#46)
HewlettPackard/c-spiffe: CONTRIBUTING.md

# Contributor guidelines

## Contributing

In order to contribute to the project, the following steps should be followed:

> 1. Fork this repository

Reviewer: ```suggestion
1. Fork the main repository
```

2. Create your feature branch
3. Commit your local changes in the feature branch
4. Push to your fork

> 5. Create a pull Request

Reviewer: ```suggestion
5. Create a pull request
```

> Pull request are going to be reviewed by mainteners, some change can be requested before the Pull Request is accepted.

Reviewer 2: ... maintainers, and changes can be ...

> Pull request are going to be reviewed by mainteners, some change can be requested before the Pull Request is accepted.

Reviewer: ```suggestion
Pull requests are going to be reviewed by maintainers, some change may be required before the Pull Request is accepted.
```

## Development setup

> For building, we recommend to build an run inside a docker container. Refer to  [BUILDING.md](BUILDING.MD) for more information.

Reviewer 2: For development purposes, we recommend building and running the project inside...

> For building, we recommend to build an run inside a docker container. Refer to  [BUILDING.md](BUILDING.MD) for more information.

Reviewer 3: ```suggestion
For building, we recommend to build and run inside a docker container. Refer to [BUILDING.md](BUILDING.MD) for more information.
```

## Conventions

> This project follows the [GNU Coding Standard](https://www.gnu.org/prep/standards/html_node/Writing-C.html) for formatting, commenting, and convetions in general. Our build system checks for formatting in the build proccess.

Reviewer 2: conventions

> This project follows the [GNU Coding Standard](https://www.gnu.org/prep/standards/html_node/Writing-C.html) for formatting, commenting, and convetions in general. Our build system checks for formatting in the build proccess.

Reviewer: ```suggestion
This project follows the [GNU Coding Standard](https://www.gnu.org/prep/standards/html_node/Writing-C.html) for formatting, commenting, and conventions in general. Our build system checks for formatting in the build process.
```

## Continuous Integration

> This project uses Github Actions as Continuous Integration platform. Refer to [CI.md](CI.MD) for mor information.

Reviewer 2: more information*

Reviewer 2: ... Actions as its Continuous Integration tool. ( or solution? platform sounds wrong)
